### DISCUSSION TITLE
Should we replace smilies with Marshmallow versions?

### DISCUSSION THREAD

**Discussion by Secret-chest** [C1710/blobmoji#124] [General]
They had more depth and emotion to them.

**Comment by igalic** [comment]
can you post examples?

**Reply by Secret-chest** [reply]
![Some comparisons](https://blog.emojipedia.org/content/images/2016/08/android-nougat-consistent-sizes.jpg)
Marshmallow is on top, Nougat is on bottom.

Fun fact: the original Android emoji were Japanese-designed

**Reply by igalic** [reply]
so much blobbier.

**Reply by Secret-chest** [reply]
If @C1710 won't agree, then it's time to fork :)

**Reply by C1710** [reply]
I personally also like these more blob-like shapes better.
However there are two reasons I wouldn't make these the default ones:
1. It would introduce some sort of inconsistency which we already have enough of
2. I can't create these myself. I'm not very good at designing stuff 😅

So for now at least I would also recommend a fork.

**Reply by Secret-chest** [reply]
Thanks.

**Comment by Secret-chest** [comment]
> I personally also like these more blob-like shapes better. However there are two reasons I wouldn't make these the default ones:
> 
> 1. It would introduce some sort of inconsistency which we already have enough of
> 2. I can't create these myself. I'm not very good at designing stuff 😅
> 
> So for now at least I would also recommend a fork.

BTW, the shapes were also reused. I can try doing it myself and seeing if we can merge.

**Comment by Secret-chest** [comment]
PR is #148
